spec/javascript/ParkShowTest.js

```diff
@@ -70,7 +71,8 @@ describe('Amusement Parks Show', () => {
           phone_number: '(413) 786-9300',
           operating_season: 'April through late December',
           website: 'https://www.sixflags.com/newengland',
-          description: "hey"
+          description: "hey",
+          reviews: []
         });
         done()
 
```

Comment:
Sorry about the nitpickiness, but each expect statement should be in its own `it` block